### ISSUE TITLE
[REQ]we need gloves and agents in web 

### ISSUE THREAD

**Issue by yichen11818** [Nereziel/cs2-WeaponPaints#200]
we need gloves and agents in web. can u update it:)

**Comment by fuzzyscn** [comment]
just use this https://github.com/LielXD/CS2-WeaponPaints-Website

**Comment by SwaggyMacro** [comment]
> just use this https://github.com/LielXD/CS2-WeaponPaints-Website

I tried this repo yesterday when I saw your reply, but it is extremely slow to access, although the repo's demo loads very quickly, I tried to modify the code but the code for this repo is not very well encapsulated, making it difficult to modify.
Then I found this repo: https://github.com/L1teD/cs2-WeaponPaints-website
But the public version of this repo doesn't support gloves.
So I made this version: https://github.com/SwaggyMacro/cs2-WeaponPaints-Web
This modified version supports gloves and it has good optimization in Chinese.
